resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/config.scala

```diff
@@ -54,17 +54,17 @@ package object config {
       .stringConf
       .createWithDefault(s"spark-executor:$sparkVersion")
 
-  private[spark] val KUBERNETES_CA_CERT_FILE =
-    ConfigBuilder("spark.kubernetes.submit.caCertFile")
+  private[spark] val KUBERNETES_APISERVER_CA_CERT_FILE =
+    ConfigBuilder("spark.kubernetes.apiserver.caCertFile")
       .doc("""
           | CA cert file for connecting to Kubernetes over SSL. This
           | file should be located on the submitting machine's disk.
         """.stripMargin)
       .stringConf
       .createOptional
 
-  private[spark] val KUBERNETES_CLIENT_KEY_FILE =
-    ConfigBuilder("spark.kubernetes.submit.clientKeyFile")
+  private[spark] val KUBERNETES_CLIENT_APISERVER_KEY_FILE =
+    ConfigBuilder("spark.kubernetes.apiserver.clientKeyFile")
```

Comment:
That naming scheme sounds good.  I forget sometimes this is all under spark.kubernetes so hopefully it's implied that clientKeyFile is probably for connections to kubernetes.

If the convention is that config in `spark.ssl` is for listening Spark services, and `spark.kubernetes` is for connecting to k8s apiserver, that seems pretty understandable.  What config space is used for connecting to a listening Spark service, e.g. when submitter talks to submitserver ?

`driversubmitserver` is a bit long, thoughts on just `submitserver` ?